adal/build.gradle

```diff
@@ -141,12 +141,13 @@ dependencies {
 
     // 'local' flavor dependencies
     localApi(project(":common")) {
-        transitive = false
+        transitive = true
     }
 
     snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '3.2.0', changing: true)
 
     // 'dist' flavor dependencies
+    //TODO: we will have to change transitive to true once common4j is published
```

Comment:
Would this work when common4j is not yet published - but is consumed by common?

Reply:
I don't know yet - will need to publish a common with the changes made here https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1338 and try it out

Reply:
Let's try it out - if we need to change it here, it's a breaking change (meaning that we probably wouldn't want to do this now).

Reply:
I guess it makes sense to just make it true right now